readme.md

```diff
@@ -22,6 +22,7 @@ randomWord();
 //=> 'microfloppies'
 ```
 
+The [underlying corpus](https://raw.githubusercontent.com/sindresorhus/word-list/master/words.txt) contains 274,925 English words.
```

Comment:
corpus => list of words

Can you also include the size in megabytes? I think that would be useful.